src/debug.c

```diff
@@ -1004,6 +1005,34 @@ NULL
     } else if (!strcasecmp(c->argv[1]->ptr, "dict-resizing") && c->argc == 3) {
         server.dict_resizing = atoi(c->argv[2]->ptr);
         addReply(c, shared.ok);
+    } else if (!strcasecmp(c->argv[1]->ptr,"script") && c->argc == 3) {
+        if (!strcasecmp(c->argv[2]->ptr,"list")) {
+            dictIterator *di = dictGetIterator(getLuaScripts());
+            dictEntry *de;
+            while ((de = dictNext(di)) != NULL) {
+                luaScript *script = dictGetVal(de);
+                sds *sha = dictGetKey(de);
+                serverLog(LL_WARNING, "SCRIPT SHA: %s\n%s", (char*)sha, (char*)script->body->ptr);
+            }
+            dictReleaseIterator(di);
+        } else if (sdslen(c->argv[2]->ptr) == 40) {
+            char funcname[43];
+            funcname[0] = 'f';
+            funcname[1] = '_';
+            memcpy(funcname+2, (char*)c->argv[2]->ptr, 40);
+            funcname[42] = '\0';
+            dictEntry *de;
+            if ((de = dictFind(getLuaScripts(), funcname+2)) == NULL) {
```

Comment:
don't need `funcname`?
```suggestion
            dictEntry *de;
            if ((de = dictFind(getLuaScripts(), c->argv[2]->ptr)) == NULL) {
```